### PR TITLE
docs: add missing `og:video` meta tag

### PR DESCRIPTION
**Pull request by Marukome0743**
# Summary
Add `og:video` meta tag to `<head>` output.

## Description
Video example was added to the `openGraph` object in `metadata` at #66613.
However this commit forgot output result.

### Improving Documentation

- [x] Run `pnpm prettier-fix` to fix formatting issues before opening the PR.
- [x] Read the Docs Contribution Guide to ensure your contribution follows the docs guidelines: https://nextjs.org/docs/community/contribution-guide

x-ref: #66613

### CODE REVIEW

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: bc6196aab123a3004ea441c73ce87e94826e5ac9

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

**Review by huozhi — APPROVED** [review]

File: docs/02-app/02-api-reference/04-functions/generate-metadata.mdx [hunk 1]
```diff
@@ -521,6 +521,9 @@ export const metadata = {
 <meta property="og:image:width" content="1800" />
 <meta property="og:image:height" content="1600" />
 <meta property="og:image:alt" content="My custom alt" />
+<meta property="og:video:url" content="https://nextjs.org/video.mp4">
```
Comment (huozhi) [inline docs/02-app/02-api-reference/04-functions/generate-metadata.mdx:524]: lint is failing
```suggestion
<meta property="og:video:url" content="https://nextjs.org/video.mp4" />
```

Comment (Marukome0743) [reply docs/02-app/02-api-reference/04-functions/generate-metadata.mdx]: Oh... Thank you for check lint.

BTW, this code is html, **not JSX**.
That's why we might use plane html tag.

The real output is
`<meta property="og:video:url" content="https://nextjs.org/video.mp4">`

However, it's not big matter on this section.

File: docs/02-app/02-api-reference/04-functions/generate-metadata.mdx [hunk 2]
```diff
@@ -521,6 +521,9 @@ export const metadata = {
 <meta property="og:image:width" content="1800" />
 <meta property="og:image:height" content="1600" />
 <meta property="og:image:alt" content="My custom alt" />
+<meta property="og:video:url" content="https://nextjs.org/video.mp4" />
```
Comment (Netail) [inline docs/02-app/02-api-reference/04-functions/generate-metadata.mdx:524]: `og:video:url` is not a valid property; https://ogp.me/#structured

Unlike `og:image`, which does have a `:url` variant, video does not

Comment (huozhi) [reply docs/02-app/02-api-reference/04-functions/generate-metadata.mdx:524]: Good catch, it should be `og:video`

Comment (Netail) [reply docs/02-app/02-api-reference/04-functions/generate-metadata.mdx:524]: As audio & video do not have a `:url` variant, should we make this consistent for image?

Comment (Marukome0743) [reply docs/02-app/02-api-reference/04-functions/generate-metadata.mdx:524]: Nice!

In addition, `og:image` examples still use `og:image:url` in docs.
We will update the below code and docs.
https://github.com/vercel/next.js/blob/2a87b33ff344c6ac4734498d9d4611d10c3aab7a/packages/next/src/lib/metadata/generate/meta.tsx#L56-L61

Comment (Netail) [reply docs/02-app/02-api-reference/04-functions/generate-metadata.mdx:524]: https://github.com/vercel/next.js/pull/69959, ahh already did :P

**Pull request merged**